packages/api/test/user.spec.js

```diff
@@ -448,7 +448,13 @@ describe('userLoginPost', function () {
     // now ensure it has desired subscription
     const gotUser = await env.db.getUser(user1Authentication.issuer, {})
     const gotSubscription = await env.subscriptions.getSubscription(
-      (await env.customers.getOrCreateForUser({ id: gotUser._id })).id
+      (
+        await env.customers.getOrCreateForUser({
+          id: gotUser._id,
+          name: 'test user',
```

Comment:
I don't think we should have `name` or `email` be required in all `getOrCreateForUser` invocations. e.g. for this test is it really needed?

Reply:
Updated.